### PR TITLE
chore(themer): add orbit-themer to repository

### PR DESCRIPTION
**Pull request by DSil**
[ORBIT-1568](https://jira.kiwi.com/browse/ORBIT-1568)
Orbit themer was moved from [its own repository](https://gitlab.skypicker.com/frontend/orbit-themer) to the monorepo.

Some components were in older versions, so a few adjustments had to be made.

Missing:
- [ ] Script to deploy the website to production
 Storybook: https://orbit-mainframev-add-orbit-themer-package.surge.sh

### CODE REVIEW

File: packages/orbit-themer/.flowconfig [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[ignore]
+<PROJECT_ROOT>/node_modules/resolve/test/resolver/malformed_package_json/package.json
```
Comment (mainframev) [inline packages/orbit-themer/.flowconfig:2]: Remove that `.flowconfig` and let's rely on the root config, add there that ignore rule

File: packages/orbit-themer/src/Tabs/ColorTab.jsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+      opened ? theme.orbit.paletteCloudDark : theme.orbit.paletteCloudLight};
+  }
+`;
+
+const StyledColorTabIcon = styled.div`
+  position: absolute;
+  top: 50%;
+  margin-top: -8px;
+  right: 12px;
+  color: #bac7d5;
```
Comment (mainframev) [inline packages/orbit-themer/src/Tabs/ColorTab.jsx:36]: I think we have a token for that value, check the palette inside design tokens ;)

File: packages/orbit-themer/src/Tabs/ColorTab.jsx [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+`;
+const StyledColorTabChildren = styled.div`
+  max-height: ${({ opened }) => (opened ? 100 : 0)};
```
Comment (mainframev) [inline packages/orbit-themer/src/Tabs/ColorTab.jsx:39]: could it be rewritten to a single destructured SC? 🙏🏻

File: packages/orbit-themer/src/index.html [hunk 4]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>Orbit Themer</title>
+    <link
```
Comment (mainframev) [inline packages/orbit-themer/src/index.html:6]: could you also add preconnect link? 🙏🏻

**Review by mainframev — CHANGES_REQUESTED** [review]

File: packages/orbit-themer/.flowconfig (hunk 1 above)
Comment (DSil) [reply packages/orbit-themer/.flowconfig:2]: We actually already have this rule there `.*/node_modules/rechoir/node_modules/resolve/test/resolver/malformed_package_json/.*` so it should be enough.

**Review by mainframev — APPROVED** [review]

File: packages/orbit-themer/package.json [hunk 5]
```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@kiwicom/orbit-themer",
+  "version": "0.0.3",
+  "description": "Design your own orbit-components theme with custom colors",
```
Comment (mainframev) [inline packages/orbit-themer/package.json:4]: could you also add `private: true`? 🙏🏻

File: packages/orbit-themer/package.json [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+  "scripts": {
+    "dev": "webpack-dev-server --mode development --open --hot --host 0.0.0.0",
+    "flow": "flow",
+    "build": "webpack --mode production"
+  },
+  "author": "Tomáš Hapl",
+  "license": "MIT",
+  "dependencies": {
+    "@kiwicom/orbit-components": "3.3.0",
+    "@kiwicom/orbit-design-tokens": "^2.0.0",
```
Comment (mainframev) [inline packages/orbit-themer/package.json:16]:
```suggestion
    "@kiwicom/orbit-design-tokens": "*",
```

File: packages/orbit-themer/package.json [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+  "main": "index.js",
+  "scripts": {
+    "dev": "webpack-dev-server --mode development --open --hot --host 0.0.0.0",
+    "flow": "flow",
+    "build": "webpack --mode production"
+  },
+  "author": "Tomáš Hapl",
+  "license": "MIT",
+  "dependencies": {
+    "@kiwicom/orbit-components": "3.3.0",
```
Comment (mainframev) [inline packages/orbit-themer/package.json:15]:
```suggestion
    "@kiwicom/orbit-components": "*",
```

**Review by mainframev — COMMENTED** [review]
forgot about these

**Pull request merged**